From 7bde6868ff24001f8b6deb8cf643d86d71978b93 Mon Sep 17 00:00:00 2001
From: Tao Bao <tbao@google.com>
Date: Tue, 28 May 2019 14:49:41 -0700
Subject: [PATCH] Remove Android.mk rules for building bootctrl.

All the targets that need the bootctrl.$(TARGET_BOARD_PLATFORM) in this
dir should build the module with Android.bp, which is also the desired
way to install the recovery variant of bootctrl. [1] has an example
change for crosshatch.

We also no longer need BOARD_IS_AUTOMOTIVE, as the module must be
explicitly included now, in order to be built with namespace-scoped
Android.bp.

[1] https://android-review.googlesource.com/c/device/google/crosshatch/+/958004

Bug: 34254109
Test: TreeHugger
Change-Id: Ib110508065f47a742acd92e03ea42901e8002e4f
---
 Android.mk | 30 ------------------------------
 1 file changed, 30 deletions(-)
 delete mode 100644 Android.mk

diff --git a/Android.mk b/Android.mk
deleted file mode 100644
index e1794d5..0000000
--- a/Android.mk
+++ /dev/null
@@ -1,30 +0,0 @@
-ifeq ($(strip $(TARGET_USES_HARDWARE_QCOM_BOOTCTRL)),true)
-# TODO:  Find a better way to separate build configs for ADP vs non-ADP devices
-ifneq ($(BOARD_IS_AUTOMOTIVE),true)
-LOCAL_PATH := $(call my-dir)
-
-# HAL Shared library for the target. Used by libhardware.
-include $(CLEAR_VARS)
-LOCAL_C_INCLUDES += $(TARGET_OUT_HEADERS)/gpt-utils/inc
-LOCAL_CFLAGS += -Wall -Werror
-LOCAL_SHARED_LIBRARIES += liblog libgptutils libcutils
-LOCAL_HEADER_LIBRARIES := libhardware_headers libsystem_headers
-LOCAL_SRC_FILES := boot_control.cpp
-LOCAL_MODULE_RELATIVE_PATH := hw
-LOCAL_MODULE := bootctrl.$(TARGET_BOARD_PLATFORM)
-LOCAL_MODULE_OWNER := qcom
-LOCAL_PROPRIETARY_MODULE := true
-include $(BUILD_SHARED_LIBRARY)
-
-# Static library for the target. Used by update_engine_sideload from recovery.
-include $(CLEAR_VARS)
-LOCAL_C_INCLUDES += $(TARGET_OUT_HEADERS)/gpt-utils/inc
-LOCAL_CFLAGS += -Wall -Werror
-LOCAL_SHARED_LIBRARIES += liblog libgptutils libcutils
-LOCAL_HEADER_LIBRARIES := libhardware_headers libsystem_headers
-LOCAL_SRC_FILES := boot_control.cpp
-LOCAL_MODULE := bootctrl.$(TARGET_BOARD_PLATFORM)
-include $(BUILD_STATIC_LIBRARY)
-
-endif
-endif
-- 
2.15.1

